frontEnd/public/roadmap.ejs

```diff
@@ -6,19 +6,63 @@
     <title>Gallery</title>
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&family=Raleway:wght@400;600&display=swap" rel="stylesheet"> 
 
+    <!-- Mapbox CDN -->
+    <script src='https://api.mapbox.com/mapbox-gl-js/v1.12.0/mapbox-gl.js'></script>
+    <link href='https://api.mapbox.com/mapbox-gl-js/v1.12.0/mapbox-gl.css' rel='stylesheet' />
+
+    <!-- Bootstrap CDN -->
```

Comment:
Do we really need Bootstrap? For responsiveness issue... I've proposed some changes. Have a look

Reply:
Yeah, I was thinking of adding two maps and positioning using Grid system. But, putting two maps increases the API requests two times. So, I dropped that idea but forgot to remove the code